vignettes/examples/linear_lasso.Rmd

```diff
@@ -0,0 +1,20 @@
+### Linear regression with LASSO prior
+
+A simple, one-variable Bayesian linear regression model using an exponential-normal prior. 
+A exponential-normal prior can be interpreted like an equivalent to frequentist LASSO.
+
+```{r linear_lasso_greta}
+# variables & priors
+int  <- variable()
+sd   <- inverse_gamma(1, 1)
+
+lambda <- gamma(1, 1)
+tau  <- exponential(0.5 * lambda**2)
```

Comment:
I'm surprised `a**b` works already in greta, I never explicitly coded it up! Might be better to use `^` for consistency and familiarity for new users. Also spaces for the style guide, so:
```r
tau  <- exponential(0.5 * lambda ^ 2)
```